### PR TITLE
Makefile.uk: Supress warnings

### PR DESCRIPTION
**Pull request by andreittr**
Add some compiler-specific warning suppression flags.

### CODE REVIEW

File: Makefile.uk [hunk 1]
```diff
@@ -89,6 +89,8 @@ LIBPYTHON3_SUPPRESS_FLAGS-y += -Wno-missing-field-initializers
 LIBPYTHON3_SUPPRESS_FLAGS-y += -Wno-implicit-fallthrough
 LIBPYTHON3_SUPPRESS_FLAGS-y += -Wno-char-subscripts
 LIBPYTHON3_SUPPRESS_FLAGS-y += -Wno-sign-compare
+LIBPYTHON3_SUPPRESS_FLAGS-$(call have_gcc) += -Wno-address
+LIBPYTHON3_SUPPRESS_FLAGS-$(call have_clang) += -Wno-tautological-pointer-compare
```
Comment (RaduNichita) [inline Makefile.uk]: I would firsly put all items related to gcc, then all items related to clang. So, I would move the second line you added 3 lines below

**Review by RaduNichita — APPROVED** [review]
All good from my side. Tested with clang-16 and gcc-13. Thanks, @andreittr 

Reviewed-by: Radu Nichita <radunichita99@gmail.com>

**Review by mariasfiraiala — APPROVED** [review]
Thanks, looks good.

Reviewed-by: Maria Sfiraiala <maria.sfiraiala@gmail.com>

**Review by razvand — APPROVED** [review]
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

**Pull request closed without merging**